src/write.rs

```diff
@@ -435,7 +434,7 @@ impl<W: Write + Seek> Write for ZipWriter<W> {
                 if let Ok(count) = write_result {
                     self.stats.update(&buf[0..count]);
                     if self.stats.bytes_written > spec::ZIP64_BYTES_THR
-                        && !self.files.last_mut().unwrap().large_file
+                        && !self.files.last_mut().unwrap().1.large_file
```

Comment:
Could make sense to add a `pub(crate)` accessor like `self.last_file()` to do this sort of thing in a future refactoring PR.
```suggestion
                        && !self.files.last().unwrap().1.large_file
```